index.html

```diff
@@ -392,5 +424,6 @@ <h3 class="info-note">
     </div>
     <script src="js/main.js"></script>
```

Comment:
Getting 404 error for main.js file
I think the file name is renamed to index.js

Reply:
Made the change please check now.

Reply:
Yes now it is working fine